src/Abc.Zebus.MessageDsl/Generator/ProtoGenerator.cs

```diff
@@ -30,11 +30,12 @@ private string Generate()
         Reset();
 
         WriteHeader();
+        var messageTree = SymbolNode.Create(Contracts);
```

Comment:
There should be a filter just like for the enums:

`SymbolNode.Create(Contracts.Messages.Where(msg => msg.Options.Proto))`